SG14/inplace_function.h

```diff
@@ -224,10 +224,11 @@ class alignas(AlignmentT) inplace_function<RetT(ArgsT...), CapacityT, AlignmentT
 		m_ManagerFctPtr = nullptr;
 	}
 
-	template<size_t OtherCapacityT>
-	void copy(const inplace_function<RetT(ArgsT...), OtherCapacityT>& other)
+	template<size_t OtherCapacityT, size_t OtherAlignment>
```

Comment:
LGTM :shipit: — but here and throughout, you might want to rename `OtherCapacityT` to just `OtherCapacity`. Looks like the existing code is inconsistent in how it names this parameter... but it's totally not a `T`ype.